### ISSUE TITLE
dependent variable hardcoded for plotting model predictions

### ISSUE THREAD

**Issue by slhogle** [jtimonen/lgpr#12]
**Environment**

```
lgpr v1.0.4
rstantools 2.0.0
rstan 2.21.1
R 4.0.2
```

**Describe the bug**
If the dependent variable in your model is named something other than `y` then the `plot_pred()` function fails

**To Reproduce**
```
testdata_002_new <- testdata_002 %>% 
  dplyr::rename(time=age, depvar=y)

fit <- lgp(depvar ~ gp(time) + zs(id)*gp(time) + zs(sex)*gp(time) + gp_vm(diseaseAge) + zs(group),
           data     = testdata_002_new,
           prior    = list(wrp = log_normal(-0.7, 0.3)), 
           iter     = 200,
           chains   = 2,
           cores = 2,
           refresh  = 50)

t <- seq(1, 120, by = 1)
x_pred <- new_x(testdata_002_new, t, x = "time", x_ns = "diseaseAge")
p <- pred(fit, x_pred, reduce = mean)
```

This works
```plot_f(fit, x = x_pred, pred = p, t_name = "time")```

This fails
```
plot_pred(fit, x = x_pred, pred = p, t_name = "time")

Error in FUN(X[[i]], ...) : object 'y' not found
```

**Expected behavior**
That the dependent variable can be named something other than `y`

Thanks!
-shane

**Comment by jtimonen** [comment]
Thanks for noticing this and for the good bug report and example code. I need to fix this.

**Comment by jtimonen** [comment]
Fixed in v1.0.5. The above code does not reproduce the bug anymore, instead works as expected.
